### ISSUE TITLE
Don't allow the player to walk off the edge of the map

### ISSUE THREAD

**Issue by robbrit** [robbrit/ossu-game-project-1#24]
Right now it's possible to walk off the edge of the map if a wall isn't there. This should not be allowed.

**Comment by Jerry8538** [comment]
I've written a method in model.py that checks if the player_sprite is out of bounds, but would it be better style to have that as a part of controller.py, where it would just not change the velocity?

**Comment by robbrit** [comment]
Nah, any sort of collision detection is related to the model, not the controller. The responsibility of the controller is to translate user actions into signals to the model or view, not to validate or constrain them.

**Comment by Jerry8538** [comment]
Alright, that makes sense, ty
